docs/gettingstarted.rst

```diff
@@ -173,7 +173,7 @@ Deploy on CERN infrastructure
 
       (reana) $ pip install reana-cluster
 
-9. Create your own ``reana-cluster.yaml``. For instance, to deploy REANA
+9. Create your own ``reana-cluster-minikube.yaml``. For instance, to deploy REANA
```

Comment:
Here it could be useful to leave `reana-cluster.yaml`, so it represents "writing your own reana-cluster.yaml" which might be deployed on something different than Minikube, what do you think?